src/engraving/rw/read400/read400.cpp

```diff
@@ -662,6 +664,30 @@ bool Read400::pasteStaff(XmlReader& e, Segment* dst, staff_idx_t dstStaff, Fract
                 }
             }
         }
+        // Set tremolo chords
+        for (TremoloTwoChord* trem : twoNoteTrems) {
+            Chord* c1 = toChord(trem->explicitParent());
+            Segment* c2Seg = c1->segment()->next();
+            while (c2Seg) {
```

Comment:
Here you can add
```
if (!c2seg->isChordRestType()) {
    continue;
}
```
for speedup. Or maybw use `segment->next(SegmentType::ChordRest)` or whatever it is, which does the same thing.